### DISCUSSION TITLE
How to check if my local `srflx` candidate is reachable from other peers

### DISCUSSION THREAD

**Discussion by navinkumarr** [coturn/coturn#1312] [Q&A]
### Problem

I am trying to establish peer to peer connection between my browser and remote webrtc application. I am trying to establish session using only `stun` candidate using `coturn` server to achieve that. 

The problem is that `p2p` connection works when I am on Hotspot network and it does not work on WiFi.

### Setup

`coturn` server is running on port `443` on `X.X.190.32`

### Findings till now

1. Remote application is successfully able to collect `srflx` candidate for itself.
2. Browser is receiving successful `srflx` candidate from the `coturn` server and shared it with the remote application using signalling server

![Screenshot 2023-11-06 at 9 03 31 PM](https://github.com/coturn/coturn/assets/5038866/4c1b71ff-59b8-4826-a314-92f7935edd38)

Even after this in `webrtc-internals` I can see that there is no `srflx-srflx` candidate pair generated on `WiFi` network.

To debug this, after initiating peer connection from client side I am trying to initiate a `udp` connection to the local machine on the `srflx` candidate port on my local machine from the remote machine but I am not getting any response.

``` bash
nc -u -n -z X.X.177.171 12474
```

**Would appreciate if I can get any pointers on how can I debug this further ?**

**Comment by chchzh1978** [comment]
Maybe, you need a signaling server
